feature/VCMS70 - Account Management Module

blackMisay opened the pull request:
Added functionalities for Account Management, and authentication.

Kindly use this updated `user` table
[user.txt](https://github.com/user-attachments/files/17589124/user.txt)

User accounts:

| Username | Password |
| ----------- | ---------- |
|    admin    |   adminR  |
|    staff1     |    staff1    |
|    dan        |    dan       |

Commit:
feature/VCMS70 - Account Management Module

lhyra13 reviewed (commented):
![462560872_1284153609183109_5610594336404924852_n](https://github.com/user-attachments/assets/23f7e415-7dd7-4dc6-a448-4a6466eeaa74)
 
the password required still appears even if a password is entered.

Commit:
feature/VCMS70 - Major bug fix for password insert and some added validations

blackMisay committed:
Merge branch 'int/INT_2024_11' into feature/VCMS70

Commit:
feature/VCMS70 - added keypress in login

blackMisay commented:
> ![462560872_1284153609183109_5610594336404924852_n](https://private-user-images.githubusercontent.com/161996378/383534011-23f7e415-7dd7-4dc6-a448-4a6466eeaa74.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA5MDg1MzAsIm5iZiI6MTczMDkwODIzMCwicGF0aCI6Ii8xNjE5OTYzNzgvMzgzNTM0MDExLTIzZjdlNDE1LTdkZDctNGRjNi1hNDQ4LTRhNjQ2NmVlYWE3NC5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTA2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEwNlQxNTUwMzBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04ZmExMjIyNTNhYzgzYTcxMjNjMzVhN2IyNzE3NzBjM2Q5MWExYzY0ZmRkNzUwZjBkNmMwYWE3OTkxYzc0ZTM5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.mMsAcMbY7iXMhTLupvZrHj75zJX0vfto-dMaRNA0IH0)
> 
> the password required still appears even if a password is entered.

Hi @lhyra13 

I pushed a new change that covers the following:
1. Bug found related to password validation during creating of user.
2. added field validations in User modal.
3. added field validations in Login.

Kindly review my changes. Thanks

Commit:
Merge branch 'feature/VCMS70' of https://github.com/blackMisay/vet-management into feature/VCMS70
